### DISCUSSION TITLE
Bad Duplicate: [webextension-polyfill]@`namespaces/manifest.d.ts` and [chrome]@`Manifest.WebExtensionManifest`

### DISCUSSION THREAD

**Discussion by Malix-Labs** [DefinitelyTyped/DefinitelyTyped#70004] [Issues with a @types package] [answered]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/db345b918c3673848d74fde71a91b130f1135282/types/webextension-polyfill/namespaces/manifest.d.ts#L31

(or

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/db345b918c3673848d74fde71a91b130f1135282/types/webextension-polyfill/namespaces/manifest.d.ts#L91
)

And

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/db345b918c3673848d74fde71a91b130f1135282/types/chrome/index.d.ts#L7455

Seems to be duplicate

[a comment by an automated account is not shown]

**Reply by Malix-Labs** [reply]
It is also about "chrome"

**Comment by Lusito** [comment, marked as answer]
Hi @Malix-off, you can either use the chrome types or the types for the webextension-polyfill, not both together. The chrome types are only for chrome and the ones for the webextension-polyfill have been generated from schema files provided by Mozilla (for the official webextension standard).

**Reply by Malix-Labs** [reply]
Okay;

Although, the chrome one have more types, and some fields are incompatible with webextension-polyfill

I think there are some inconsistency too between MV2 and MV3 specs
